tests/framework-tests/core/js/ModelTransformationTests.js

```diff
@@ -1026,18 +1031,65 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             type: "fluid.transforms.numberToString",
             inputPath: "hundred"
         },
-        method: "assertValue",
         expected: fluid.tests.transforms.source.hundredInString,
         expectedInputPaths: [ "hundred" ]
-
+    }, {
+        message: "numberToString() converts integers - with scale.",
+        transformWrap: true,
+        transform: {
+            type: "fluid.transforms.numberToString",
+            inputPath: "hundred",
+            scale: 2
+        },
+        expected: fluid.tests.transforms.source.hundredInString,
+        expectedInputPaths: [ "hundred" ]
     }, {
         message: "numberToString() converts float values.",
         transformWrap: true,
         transform: {
             type: "fluid.transforms.numberToString",
             inputPath: "floatyHighy"
         },
-        method: "assertValue",
+        expected: fluid.tests.transforms.source.floatInString,
+        expectedInputPaths: [ "floatyHighy" ]
+    }, {
+        message: "numberToString() converts float values - with scale = 2.",
```

Comment:
I think we can remove the terminal full stops from all of these messages as being somewhat confusing. Messages are not sentences after all :)